Reportedly/Models/Report.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Report.swift
+//  Reportedly
+//
+//  Created by Mikey Pham on 12/9/20.
+//  Copyright © 2020 NimbleHQ. All rights reserved.
+//
+
+import Foundation
+
+
+struct Report: Decodable {
+    
+    // MARK: - Decodable Enums
+    enum Category: String, Decodable {
+        case swift, combine, debugging, xcode
+    }
+
+    enum CodingKeys: String, CodingKey {
+        case id, type, attributes
+    }
+    
+    var id: String = ""
+    var type: String = ""
+    var attributes: ReportAttributes
+    
+    var obstacles: String {
+        attributes.obstaclesToday
+    }
+    var tasks: String {
+        attributes.taskToday
+    }
+    var date: Date {
+        attributes.user.createdAt.toDate(withFormat: DateFormat.yyyy_MM_dd_T_HH_mm_ss_SSSZ)
+    }
+    
+    init(from decoder: Decoder) throws {
```

Comment:
I think `Decodable ` with init this struct for free, then we don't have to init it if we don't have any custom here. Could you please give more insight that why do you want to set empty string for id and type?  🙏

Reply:
yeah, back then I have some parsing issues for this Report object and finally solve it with type updating, but not remove the manual parser yet, let me remove 🙏